lib/blpapi-wrapper.ts

```diff
@@ -376,8 +396,7 @@ export class Session extends events.EventEmitter {
             }
             var uri = '//blp/apiauth';
             var requestName = 'AuthorizationRequest';
-            var isAuthRequest = true;
-            this.doRequest(uri, requestName, request, callback, isAuthRequest);
+            this.doRequest(uri, requestName, request, performAuthorizeUser, callback);
```

Comment:
You could `this.session.authorizeUser.bind(this.session, request)`.  Then the `performAuthorizeUser` may be removed.